### PR TITLE
add runtest to sdk suites

### PR DESCRIPTION
**Pull request by salsal97**

### CODE REVIEW

**Comment by salsal97** [comment]
Successful run here: https://oe-jenkins-dev.westeurope.cloudapp.azure.com/blue/organizations/jenkins/Mystikos%2FStandalone-Pipelines%2FAzure-SDK-Tests-Pipeline/detail/Azure-SDK-Tests-Pipeline/979/pipeline/56

File: solutions/dotnet_azure_sdk/Makefile [hunk 1]
```diff
@@ -10,6 +10,12 @@ DIRS += KeyVault_Certificates
 REDEFINE_TESTS=1
 include $(TOP)/rules.mak
 
+ifdef RUN_AZURE_TESTS
```
Comment (jxyang) [inline solutions/dotnet_azure_sdk/Makefile:13]: Under what circumstances we want `RUNTEST` to be empty?

**Review by CyanDevs — APPROVED** [review]
LGTM

File: tests/azure-sdk-for-cpp/Makefile [hunk 2]
```diff
@@ -48,4 +55,4 @@ clean:
 	rm -rf $(APPDIR) $(ROOTFS)
 
 get-executables:
-	find $(RUNTEST) $(MYST_EXEC) $(OPTS) $(ROOTFS) /azure-sdk-for-cpp/build/sdk -type f -executable -print | grep test
+	find $(MYST_EXEC) $(OPTS) $(ROOTFS) /azure-sdk-for-cpp/build/sdk -type f -executable -print | grep test
```
Comment (jxyang) [inline tests/azure-sdk-for-cpp/Makefile:58]: Seems this target is not used anymore? What's the purpose of it?

File: solutions/dotnet_azure_sdk/Makefile (hunk 1 above)
Comment (salsal97) [reply solutions/dotnet_azure_sdk/Makefile:13]: When we are running any of the tests independently, in that case runtest would be empty

File: tests/azure-sdk-for-cpp/Makefile (hunk 2 above)
Comment (CyanDevs) [reply tests/azure-sdk-for-cpp/Makefile:58]: I wondered this myself. There's no mention of it but I guess it is kept around as a convenience for finding test executables

Comment (salsal97) [reply tests/azure-sdk-for-cpp/Makefile:58]: This target is used when we extract the binaries from the docker container, mostly it is kept for convenience

Comment (jxyang) [reply tests/azure-sdk-for-cpp/Makefile:58]: We can find the tests from appdir without invoking mystikos. In addition, I suspect the command should be:
```
$(MYST_EXEC) $(OPTS) $(ROOTFS) find /azure-sdk-for-cpp/build/sdk -type f -executable -print |  grep test
```

Comment (CyanDevs) [reply tests/azure-sdk-for-cpp/Makefile:58]: Good catch Xuejun

**Pull request merged**